### PR TITLE
Use new query model to get audit logs

### PR DESCRIPTION
**Pull request by wy65701436**
leverage the query builder to build query, remove the old style query string

Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: src/server/v2.0/handler/auditlog.go [hunk 1]
```diff
@@ -26,28 +25,9 @@ func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAud
 	// if !a.HasPermission(ctx, rbac.ActionList, rbac.ResourceLog) {
```
Comment (ywk253100) [inline src/server/v2.0/handler/auditlog.go]: Will we do the permission check in another PR?

Comment (wy65701436) [reply src/server/v2.0/handler/auditlog.go]: yes  to resolve ToDo items
